viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -735,7 +735,7 @@ export class Cognite3DViewer {
       preMultipliedRotation
     );
 
-    return { image360Entities };
+    return image360Entities;
```

Comment:
`image360Entities` should probably be renamed to `image360Collection` now right?